zeppelin-interpreter/src/main/java/org/apache/zeppelin/helium/HeliumPackage.java

```diff
@@ -18,13 +18,16 @@
 
 import org.apache.zeppelin.annotation.Experimental;
 
+import java.util.Map;
+
 /**
  * Helium package definition
  */
 @Experimental
 public class HeliumPackage {
   private HeliumType type;
   private String name;           // user friendly name of this application
+  private String version;
```

Comment:
> it's hard to get version field when package is local. In this case artifact doens't include version value at all.

When package is local, isn't `artifact` (path of the package, in this case) an enough information that Zeppelin needs? I think having `version` when package is local may introduce some confusions & errors.

 - `version` of local package and online package can be the same. It happens when developing helium package and publishing npm. i.e. after publishing package to npm, local package in development mode and remote package version name will be the same.
 - There're possibility of human error that multiple local package is deployed with the name `version` field.

both cases, result is undefined.
However, if we just treat `artifact` as a version, it's not only a one less field but also no possible confusion or human error. 

what do you think?